src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrder.cs

```diff
@@ -1,57 +1,57 @@
-﻿using System;
-using System.Net.Http;
-using System.Threading.Tasks;
-using SwedbankPay.Sdk.Extensions;
-
-namespace SwedbankPay.Sdk.PaymentOrders
-{
-    public class PaymentOrder
-    {
-        private PaymentOrder(PaymentOrderResponse paymentOrderResponse,
-                             HttpClient client)
-        {
-            PaymentOrderResponse = paymentOrderResponse.PaymentOrderResponseObject;
-            var operations = new PaymentOrderOperations(paymentOrderResponse.Operations, client);
-            Operations = operations;
-        }
-
-
-        public PaymentOrderOperations Operations { get; }
-        public PaymentOrderResponseObject PaymentOrderResponse { get; }
-
-
-        internal static async Task<PaymentOrder> Create(PaymentOrderRequest paymentOrderRequest,
-                                                        HttpClient client,
-                                                        string paymentOrderExpand)
-        {
-            var url = new Uri($"/psp/paymentorders{paymentOrderExpand}", UriKind.Relative);
-            
-            var paymentOrderResponseContainer = await client.PostAsJsonAsync<PaymentOrderResponse>(url, paymentOrderRequest);
-
-            return new PaymentOrder(paymentOrderResponseContainer, client);
-        }
-
-
-        /// <summary>
-        ///     Gets the payment
-        /// </summary>
-        /// <param name="id"></param>
-        /// <param name="client"></param>
-        /// <param name="paymentOrderExpand"></param>
-        /// <exception cref="ArgumentNullException"></exception>
-        /// <exception cref="InvalidOperationException"></exception>
-        /// <exception cref="System.Net.Http.HttpRequestException"></exception>
-        /// <exception cref="Exceptions.HttpResponseException"></exception>
-        /// <returns></returns>
-        internal static async Task<PaymentOrder> Get(Uri id, HttpClient client, string paymentOrderExpand)
-        {
-            var url = !string.IsNullOrWhiteSpace(paymentOrderExpand)
-                ? new Uri(id.OriginalString + paymentOrderExpand, UriKind.RelativeOrAbsolute)
-                : id;
-
-            var paymentOrderResponseContainer = await client.GetAsJsonAsync<PaymentOrderResponse>(url);
-
-            return new PaymentOrder(paymentOrderResponseContainer, client);
-        }
-    }
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using SwedbankPay.Sdk.Extensions;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class PaymentOrder : IPaymentOrder

```

Comment:
Codacy found an issue: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)